### PR TITLE
Add smart wallet address in operator response

### PR DESCRIPTION
**Pull request by MusabShakeel576**
## Proposed changes

Add smart wallet address in operator response

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Refactor or a modification (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: apps/charge-accounts-service/src/operators/operators.service.ts [hunk 1]
```diff
@@ -101,20 +106,31 @@ export class OperatorsService {
...
+        apiKey: apiKeyInfo.publicKey
       }
-      await this.createOperatorWallet(user, predictedWallet)
+      const wallet = await this.createOperatorWallet(user, predictedWallet)
       await this.addAddressToOperatorsWebhook(predictedWallet)
       await this.addAddressToTokenReceiveWebhook(predictedWallet)
       this.googleFormSubmit(createOperatorUserDto)
       this.analyticsService.trackEvent('New Operator Created', { ...eventData }, { user_id: user?.auth0Id })
-      return this.constructUserProjectResponse(user, projectObject, publicKey.publicKey, secretKey, sponsorId)
+      const project = {
```
Comment (NikolayRn) [inline apps/charge-accounts-service/src/operators/operators.service.ts:122]: Please create interface for project

File: apps/charge-accounts-service/src/operators/operators.service.ts [hunk 2]
```diff
@@ -160,25 +176,30 @@ export class OperatorsService {
     if (!operatorWalletCreationResult) {
       throw new HttpException('Failed to create operator wallet', HttpStatus.INTERNAL_SERVER_ERROR)
     }
+    return operatorWalletCreationResult
   }
 
-  private constructUserProjectResponse (user: any, projectObject: any, publicKey: string, secretKey: string, sponsorId: string) {
-    // Constructs the response object from the created entities
+  private constructUserProjectResponse (user: any, project: any, wallet: any, secretKey?: string) {
+    // Constructs the response object from the entities
```
Comment (NikolayRn) [inline apps/charge-accounts-service/src/operators/operators.service.ts:183]: Create interface also for this function please

**Review by NikolayRn — CHANGES_REQUESTED** [review]
Please see the comments @MusabShakeel576

File: apps/charge-accounts-service/src/operators/operators.service.ts (hunk 1 above)
Comment (MusabShakeel576) [reply apps/charge-accounts-service/src/operators/operators.service.ts]: @NikolayRn updated

File: apps/charge-accounts-service/src/operators/operators.service.ts (hunk 2 above)
Comment (MusabShakeel576) [reply apps/charge-accounts-service/src/operators/operators.service.ts:186]: @NikolayRn updated

**Review by NikolayRn — APPROVED** [review]
LGTM

**Pull request merged**